cryptoki/src/types/object.rs

```diff
@@ -258,6 +265,7 @@ impl Attribute {
             | Attribute::Wrap(_) => 1,
             Attribute::Base(_) => 1,
             Attribute::Class(_) => std::mem::size_of::<CK_OBJECT_CLASS>(),
+            Attribute::EcPoint(bytes) => std::mem::size_of::<u8>() * bytes.len(),
```

Comment:
I believe just `bytes.len()` would give you the same result

Reply:
Understood!

I see I copied a fragment from `Value` variant but the rest of them use just `bytes.len()`:

>            Attribute::Modulus(bytes) => bytes.len(),
>            Attribute::Value(bytes) => std::mem::size_of::<u8>() * bytes.len(),

Reply:
Ok, fixed both of them. Thanks for your review & time @hug-dev !